### DISCUSSION TITLE
keybind to back off indentation

### DISCUSSION THREAD

**Discussion by igotfr** [lapce/lapce#1289] [Q&A] [answered]
What's the keybind to back off indentation?

**Comment by vlanse** [comment, marked as answer]
do you mean these ?
<img width="519" alt="Screenshot 2023-01-26 at 21 09 31" src="https://user-images.githubusercontent.com/17192647/214915527-74fa5148-59c1-44b4-993b-b8279fd2f2c4.png">

**Reply by igotfr** [reply]
I'm used to Shift + Tab
